SPECS-EXTENDED/rubygem-deep_merge/rubygem-deep_merge.spec

```diff
@@ -8,29 +8,30 @@ Version:        1.2.1
 Release:        1%{?dist}
 Group:          Development/Languages
 License:        MIT
-Vendor:		Microsoft Corporation
+Vendor:         Microsoft Corporation
 Distribution:	Mariner
-URL:            https://rubygems.org/gems/%{gem_name}/versions/%{version}
-Source0:        https://rubygems.org/downloads/%{gem_name}-%{version}.gem
+URL:            https://github.com/danielsdeleo/deep_merge
```

Comment:
Last one, I promise: please update the `Release` tag and changelog where that's not done, yet.